lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.scss

```diff
@@ -1,7 +1,7 @@
 /* stylelint-disable no-descending-specificity */
 $content-node-selector-thumbnail-width: 35px !default;
 
-.adf-search-results-label {
+h2.adf-search-results-label {
```

Comment:
Do we need to specify h2 selector here? Wouldn't class be enough?

Reply:
Yes, adding h2 is required. Please have a look at below screenshot: 
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/115619721/200591408-52f3145a-1e23-4ce9-9e40-9b449b61da6e.png">
If you check at CSS panel then you see that when I removed h2 from first rule for adf-search-results-label then all css are greyed out and css for that header are broken. 
The reason is the second rule which is: 
```css
.mat-toolbar, .mat-toolbar h1, .mat-toolbar h2, .mat-toolbar h3, .mat-toolbar h4, .mat-toolbar h5, .mat-toolbar h6 {
    font: 500 20px / 32px Muli, Roboto, "Helvetica Neue", sans-serif;
    letter-spacing: normal;
    margin: 0;
}
```
You see that second rule includes h2 together with class name so to make equal/higher priority for my rule I needed to include h2 tag in my rule.

Reply:
Ok sure fine for me in this case :)